cmd/sealer/cmd/cluster/run.go

```diff
@@ -397,6 +410,13 @@ func installApplication(appImageName string, envs []string, app *v2.Application,
 		}
 	}()
 
+	for _, info := range imageMountInfo {
+		err = v2App.FileProcess(info.MountDir)
+		if err != nil {
+			return err
```

Comment:
use errors.Wrapf(err,"xxx") here and add more info to facilitate the precise location of the problem